homeassistant/components/media_player/reproduce_state.py

```diff
@@ -52,45 +58,75 @@ async def call_service(service: str, keys: Iterable) -> None:
         data = {"entity_id": state.entity_id}
         for key in keys:
             if key in state.attributes:
-                data[key] = state.attributes[key]
-
+                if key == ATTR_MEDIA_POSITION:
```

Comment:
I suggest we leave support for this attribute out of this PR.

Reply:
Ok. I've removed this code.